zkevm-circuits/src/keccak_circuit/util.rs

```diff
@@ -210,12 +210,18 @@ pub(crate) fn get_degree() -> usize {
 }
 
 /// Returns how many bits we can process in a single lookup given the range of
-/// values the bit can have and the height of the circuit.
-pub(crate) fn get_num_bits_per_lookup(range: usize) -> usize {
+/// values the bit can have and the height of the circuit (via KECCAK_DEGREE).
+pub fn get_num_bits_per_lookup(range: usize) -> usize {
+    let log_height = get_degree();
+    get_num_bits_per_lookup_impl(range, log_height)
+}
+
+// Implementation of the above without environment dependency.
+pub fn get_num_bits_per_lookup_impl(range: usize, log_height: usize) -> usize {
```

Comment:
Done.

These `(crate)` were added after the downstream PR. They do nothing since the module is private but I’ll add it for consistency with the surroundings.